types/v56/nsxt_types.go

```diff
@@ -255,7 +255,7 @@ type OpenApiOrgVdcNetworkDhcpPools struct {
 type NsxtFirewallGroup struct {
 	// ID contains Firewall Group ID (URN format)
 	// e.g. urn:vcloud:firewallGroup:d7f4e0b4-b83f-4a07-9f22-d242c9c0987a
-	ID string `json:"id"`
+	ID string `json:"id,omitempty"`
```

Comment:
It's OK to omit ID?

Reply:
Yes. In general, we usually put `omitempty` for `ID` fields. The reason is - during a "Create" operation we don't have ID and it is better not to send the field at all than send an empty string. There were some cases where API even breaks getting an `ID` field, but that is not this particular case. I just saw that I left it some time ago and thought to align it - if there was any problem - tests would catch it.